website/content/docs/commands/server/members.mdx

```diff
@@ -57,3 +61,40 @@ server-1.global  10.0.0.8   4648  alive   true    2         3             1.3.0
 server-2.global  10.0.0.9   4648  alive   false   2         3             1.3.0  dc1         global  id=04594bee-fec9-4cec-f308-eebe82025ae7,dc=dc1,expect=3,rpc_addr=10.0.0.9,raft_vsn=3,port=4647,role=nomad,region=global,build=1.3.0
 server-3.global  10.0.0.10  4648  alive   false   2         3             1.3.0  dc1         global  region=global,dc=dc1,rpc_addr=10.0.0.10,raft_vsn=3,build=1.3.0,expect=3,id=59542f6c-fb0e-50f1-4c9f-98bb593e9fe8,role=nomad,port=4647
 ```
+
+The `-json` flag can be used to get the latest server members information in json format:
+
+```shell-session
+$ nomad server members -json
+ [
+    {
+      "Name": "bacon-mac.global",
+      "Addr": "127.0.0.1",
+      "Port": 4648,
+      "Tags": {
+        "mvn": "1",
+        "build": "0.5.5dev",
+        "port": "4647",
+        "bootstrap": "1",
+        "role": "nomad",
+        "region": "global",
+        "dc": "dc1",
+        "vsn": "1"
+      },
+      "Status": "alive",
+      "ProtocolMin": 1,
+      "ProtocolMax": 5,
+      "ProtocolCur": 2,
+      "DelegateMin": 2,
+      "DelegateMax": 4,
+      "DelegateCur": 4
+    }
+  ]
```

Comment:
suggestion to use slightly more recent output as it has additional information populated:
```suggestion
[
    {
        "Addr": "127.0.0.1",
        "DelegateCur": 4,
        "DelegateMax": 5,
        "DelegateMin": 2,
        "Name": "bacon-mac.global",
        "Port": 4648,
        "ProtocolCur": 2,
        "ProtocolMax": 5,
        "ProtocolMin": 1,
        "Status": "alive",
        "Tags": {
            "build": "1.5.0",
            "region": "global",
            "rpc_addr": "127.0.0.1",
            "role": "nomad",
            "raft_vsn": "3",
            "bootstrap": "1",
            "expect": "1",
            "dc": "dc1",
            "port": "4647",
            "revision": "fc40c491cacec3d8ec3f2f98cd82b9068a50797c",
            "vsn": "1",
            "id": "354fd9a8-4228-ca8e-8b93-3cf002514910"
        }
    }
]
```